pages/common/telnet.md

```diff
@@ -1,19 +1,19 @@
 # telnet
 
-> Telnet is used to connect to a specified port of a host.
+> Connect to a specified port of a host using the telnet protocol.
 
 - Telnet to a certain port:
 
 `telnet  {{ip_address}} {{port}}`
 
-- To exit a telnet session:
+- Exit a telnet session:
 
 `quit`
 
-- Default escape character:
+- Emit the default escape character:
 
 `CTRL + ]`
```

Comment:
It should, but that makes it hard to read. Shall we adopt a convention of adding spaces around the + sign? That sounds reasonable and in line with the project's ethos (prioritizing readability over compactness).